proca/lib/proca_web/schema/user_types.ex

```diff
@@ -94,7 +94,7 @@ defmodule ProcaWeb.Schema.UserTypes do
 
       load(:org, by: [name: :org_name])
       determine_auth(for: :org)
-      allow([:change_org_settings])
+      allow([:change_org_users])
```

Comment:
fix unrelated previously existing bug?

Reply:
ok, make sense to comment it out so we know?
it might make sense to alter the default role when inviting someone (to be campaigner, instead of owner)